app/piiremover.py

```diff
@@ -26,28 +25,30 @@ def pii_remover(
 
     Sources:
 
-    Passports: http://regexlib.com/REDetails.aspx?regexp_id=2390
-    NHS and short passport numbers will be caught be credit cards regex.
-
-    '''
-
-    if passports:
-        x = sub(passports, replace, x)
+    Passport: http://regexlib.com/REDetails.aspx?regexp_id=2390
+    Email: https://emailregex.com/
     
-    if dates:
-        x = sub(dates, replace, x)
+    Note: NHS and short passport numbers will be caught by credit cards regex
```

Comment:
Yep - in fact I removed the credit card regex at some point. So this needs some additional thought.